app/utils/transaction_view.js

```diff
@@ -33,15 +33,16 @@ module.exports = {
     connectorData.pageSizeLinks = getPageSizeLinks(connectorData)
 
     connectorData.cardBrands = lodash.uniqBy(allCards.card_types, 'brand')
-      .map((card) => {
-        let value = {}
-        value.text = card.label
-        if (card.brand === filtersResult.brand) {
-          value.selected = true
+      .map(card => {
+        return {
+          value: card.brand,
+          text: card.label === 'Jcb' ? card.label.toUpperCase() : card.label,
```

Comment:
Another strange one with the treatment of jcb/Jcb/JCB. Are we just trying to work around it being stored in the database in the wrong format (or something).